src/Model/Gpio.php

```diff
@@ -9,6 +9,9 @@
  */
 class Gpio
 {
+    const OUT	= 'OUT';
+    const IN	= 'IN';
+    const PWM	= 'PWM';
     const READ  = 'READ';
     const WRITE = 'WRITE';
 
```

Comment:
READ & WRITE constants need to be removed. They will not be needed any longer as they are replaced with the new IN & OUT ones.